test/chirps.test.js

```diff
@@ -0,0 +1,68 @@
+require('dotenv').config();
+require('../lib/utils/connect')();
+
+const mongoose = require('mongoose');
+const app = require('../lib/app');
+const request = require('supertest');
+const seedData = require('./seedData');
+const Chirp = require('../lib/models/Chirp');
+
+jest.mock('../lib/middleware/ensureAuth');
+jest.mock('../lib/services/auth.js');
+
+const createChirp = (handle, text = 'some text') => {
+  return Chirp.create({ handle, text })
+} 
+
+describe('chirps routes', () => {
+  beforeEach(done => {
+    mongoose.connection.dropDatabase((done()));
```

Comment:
```suggestion
    mongoose.connection.dropDatabase(done);
```
I found the tests results were unreliable -- sometimes failing, sometimes async timeouts. Try this syntax.